.env.sample

```diff
@@ -0,0 +1,42 @@
+# General Project Settings
+COMPOSE_PROJECT_NAME=app # Name of the Docker compose project, used to namespace container names
+ENVIRONMENT=dev # Deployment environment: can be one of local, staging, or production
+
+# OpenAI Settings
+WORKDIR=/ # Default working directory for the application
+
+# FastAPI Settings
+FASTAPI_HOST=host.docker.internal # The hostname where the FastAPI server is accessible, typically the Docker host
+FASTAPI_PORT=6001 # The port on which the FastAPI server listens (6001 is chosen to avoid conflicts on Macs)
+
+# Security Settings
+SECRET_KEY=0c0794abb399d2908c7693719f0539d5b8efed88b68f8fc2b59ba6b289a93978 # A secret key used for encryption, token generation, etc.
+ALGORITHM=HS256 # The algorithm used for encoding or decoding JWT tokens
+ACCESS_TOKEN_EXPIRE_MINUTES=30 # The expiration time in minutes for the JWT access token
+
+# PostgreSQL Database Settings
+POSTGRES_HOST= # The hostname or IP address of the PostgreSQL server
+POSTGRES_PORT= # The port number on which the PostgreSQL server is running
+POSTGRES_USER= # The username used to connect to the PostgreSQL database
+POSTGRES_PASSWORD= # The password used to connect to the PostgreSQL database
+POSTGRES_DB= # The name of the database to connect to on the PostgreSQL server
+
+# VectorDB Settings (Placeholder for actual settings)
+# Additional database or service settings can be added here
+
+# Mail Configuration
+MAIL_USERNAME= # Username for the email service account
+MAIL_PASSWORD= # Password for the email service account
+MAIL_FROM= # The email address that will appear in the "From" field of sent emails
+MAIL_PORT=465 # Port number for the email server (465 is commonly used for SSL)
+MAIL_SERVER= # Hostname or IP address of the email server
+MAIL_STARTTLS=FALSE # Specifies whether to use StartTLS (use TRUE to enable)
+MAIL_SSL_TLS=TRUE # Specifies whether to use SSL/TLS for email encryption (use TRUE to enable)
+
+# Traefik Configuration
+LETS_ENCRYPT_EMAIL= # Email address to receive Let's Encrypt notifications for SSL certificate management
+TRAEFIK_DASHBOARD_URL= # URL to access the Traefik dashboard
+TRAEFIK_DASHBOARD_USERNAME= # Username for Traefik dashboard authentication
+TRAEFIK_DASHBOARD_PASSWORD= # Hashed password for Traefik dashboard authentication
+TRAEFIK_BACKEND_URL= # URL where the backend service will be accessible
+TRAEFIK_FRONTEND_URL= # URL where the frontend service will be accessible
```

Comment:
Same as backend url, I have no idea what this is and how to figure this out.

Reply:
@kirill-andr Backend URL is the url where you access the docs of the API